src/NumericalAlgorithms/LinearSolver/Tags.hpp

```diff
@@ -73,7 +73,7 @@ struct Residual : db::PrefixTag, db::SimpleTag {
 
 /*!
  * \ingroup LinearSolverGroup
- * \brief The magnitude square of the residual \f$\langle r,r\rangle_A\f$ w.r.t.
+ * \brief The magnitude square of the residual \f$\langle r,r\rangle\f$ w.r.t.
```

Comment:
Should this be a compute item, since it no longer depends on `A`?

Reply:
Good idea, but the residual magnitude is a global quantity so it is typically stored in another parallel component, e.g. a singleton, as the result of a global reduction.